config/default-search.js

```diff
@@ -1 +1,19 @@
-module.exports = {};
+const staticHost = 'https://addons-admin.cdn.mozilla.net';
+
+module.exports = {
+  staticHost,
+
+  CSP: {
+    directives: {
+      scriptSrc: [
+        staticHost,
+      ],
+      styleSrc: [staticHost],
+      imgSrc: [
+        "'self'",
+        staticHost,
```

Comment:
Actually imgSrc will need the amo CDN host as well in each case for any add-on images linked in the search results. We can add that later though.